src/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+class TodoList extends React.PureComponent {
+  render() {
+    return (
+      <ul className="todo-list">
+        {this.props.todos.map(todoItem => (
+          <TodoItem
+            todo={todoItem}
+            completed={this.props.completed}
+            deleteItem={this.props.deleteItem}
+          />
```

Comment:
where is the `key`? it should be written here

Reply:
solved